backend/api/appsettings.Development.json

```diff
@@ -1,7 +1,15 @@
 {
   "AzureAd": {
     "ClientId": "ea4c7b92-47b3-45fb-bd25-a8070f0c495c",
-    "ClientSecret": "Fill in here or preferrably in ASP.NET Secret Manager"
+    "ClientSecret": "Fill in here or preferably in ASP.NET Secret Manager"
+  },
+  "KeyVault": {
+    "VaultUri": "https://eqrobotkeyvault.vault.azure.net/"
+  },
+  "Echo": {
+    "EchoClientId": "bf0b2569-e09c-42f0-8095-5a52a873eb7b",
+    "EchoAppScope": ".default",
+    "EchoApiUrl": "https://echohubapi.equinor.com/api"

```

Comment:
Dont think prefixing with Echo is necessary, but I have no strong opinion about this, agreed it may increase readability in some cases

Reply:
Fair, since it is within the `Echo` section.